### PR TITLE
[resto druid] Added support for T21.

### PR DESCRIPTION
**Pull request by kfinch**
T21 support for Resto Druids, using PTR logs to check my work. I also fixed some messed up dates in the changelog -_-

### CODE REVIEW

File: src/Parser/RestoDruid/CombatLogParser.js [hunk 1]
```diff
@@ -22,10 +22,13 @@ import Sephuz from './Modules/Legendaries/Sephuz';
 import DarkTitanAdvice from './Modules/Legendaries/DarkTitanAdvice';
 import EssenceOfInfusion from './Modules/Legendaries/EssenceOfInfusion';
 import Tearstone from './Modules/Legendaries/Tearstone';
-import T20 from './Modules/Legendaries/T20';
-import T19_2Set from './Modules/Legendaries/T19_2Set';
 import DarkmoonDeckPromises from './Modules/Legendaries/DarkmoonDeckPromises';
 
+import T19_2Set from './Modules/Legendaries/T19_2Set';
```
Comment (rp4rk) [inline src/Parser/RestoDruid/CombatLogParser.js]: 👍

File: src/Parser/RestoDruid/CombatLogParser.js [hunk 2]
```diff
@@ -104,8 +107,10 @@ class CombatLogParser extends CoreCombatLogParser {
     darkTitanAdvice: DarkTitanAdvice,
     essenceOfInfusion: EssenceOfInfusion,
     tearstone: Tearstone,
-    t20: T20,
     t19_2set: T19_2Set,
+    t20: T20,
+    t21_2set: T21_2Set,
```
Comment (rp4rk) [inline src/Parser/RestoDruid/CombatLogParser.js]: I think this should be renamed `t21TwoSet` etc, snake case is creeping in here.

Comment (kfinch) [reply src/Parser/RestoDruid/CombatLogParser.js]: All the other set bonus modules are named in this fashion, so I don't think it makes sense to change unless you want a broader refactor. Personally I prefer snake case to '2Two'.

**Comment by kfinch** [comment]
For testing: Jx41dmtFw92vGT3n

File: src/Parser/RestoDruid/CombatLogParser.js (hunk 2 above)
Comment (MartijnHols) [reply src/Parser/RestoDruid/CombatLogParser.js]: Naming that I've used to keep it somewhat consistent and readable was `Tier21_4set`.

Comment (kfinch) [reply src/Parser/RestoDruid/CombatLogParser.js]: So change it or keep it as is?

**Comment by MartijnHols** [comment]
Thanks!

**Pull request merged**